__tests__/app.test.js

```diff
@@ -375,4 +376,78 @@ describe("GET /api/articles/:article_id/comments", () => {
         });
     });
   })
+})
+
+describe("POST /api/articles/:article_id/comments", () => {
+  describe("STATUS 201", () => {
+    test("Responds with 201 and returns created comment", () => {
+      const newComment = { username: "butter_bridge", body: "some text"};
+      return request(app)
+        .post("/api/articles/1/comments")
+        .send(newComment)
+        .expect(201)
+        .then(({ body }) => {
+          expect(body.addedComment).toEqual({
+            comment_id: 19,
+            body: "some text",
+            article_id: 1,
+            author: "butter_bridge",
+            votes: 0,
+            created_at: expect.any(String),
+          });
+        });
+    });
+  });
+  describe("STATUS 400s", () => {
+    test("Status 400 - invalid id", () => {
+      const newComment = { username: "butter_bridge", body: "some text"}
+      return request(app)
+        .post("/api/articles/invalidId/comments")
+        .send(newComment)
+        .expect(400)
+        .then(({body}) => {
+          expect(body.msg).toBe("Invalid input data type");
+        });
+    });
+    test("STATUS 404 - article doesn't exist", () => {
+      const newComment = { username: "butter_bridge", body: "some text"}
+      return request(app)
+        .post("/api/articles/999/comments")
+        .send(newComment)
+        .expect(404)
+        .then(({body}) => {
+          expect(body.msg).toBe("Article not found!");
+        });
+    });
+    // test.only("STATUS 400 - wrong data type", () => {
```

Comment:
ideally you'll have decided if you want the code or not before submitting a PR: decide and keep/destroy as you wish